projects/COVID19_FoldingAtHome.yaml

```diff
@@ -4,7 +4,7 @@ FieldOfScience: Biochemistry
 ID: '661'
 Name: COVID19_FoldingAtHome
 Organization: University of Nebraska-Lincoln
```

Comment:
Should we also update the org?

Reply:
yes! Did we ever agree on the "Multi-Institutional" option? See: https://foldingathome.org/about/the-foldinghome-consortium/

Reply:
Just put it now, we can fix it later.  Lets get this merged.